app/components/standup_meeting_group/joined_card_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= tag.div(id: dom_id(standup_meeting_group), class: "card card-bordered w-72") do %>
+  <div class="card-body">
```

Comment:
while the mocks don't have to be pixel perfect, they should mirror the visual hierarchy that are set out in the wireframes. 

Specifically, this means that you should have two columns and the leave button should look like a link. I'm also not seeing the CTA that shows whether or not you've checked in for the group for today's date? I'd be alright pushing that to another PR if you add a different issue after closing, but I think that's important.
Also, how do these look on mobile?

Reply:
Hey @afogel Thanks for the review!  I added a couple new screen recordings. What do you think? 

We still need to make those joined card components to be responsible on mobile, but this is what we have so far.